collectors/collector-json.js

```diff
@@ -1,7 +1,10 @@
-const collector_json = function (data, params) {
+const collector_json = function (datas, params) {
     // convert JSON data string to object form
-    if (typeof data === 'string' || data instanceof String) {
-        data = JSON.parse(data);
+
+    for (let [i, data] of datas.entries()) {
```

Comment:
is datas no longer an array here?

Reply:
entries() transforms the array into tuples containing an index and data